docs/api.rst

```diff
@@ -12475,3 +12475,111 @@ Clear Icons/Avatars Cache
    :statuscode 200: The cache items were cleared successfully.
    :statuscode 400: Entries provided do not match the ``cache_type`` specified. Invalid ``cache_type`` provided.
    :statuscode 500: Internal rotki error
+
+
+Event Mappings
+================
+
+.. http:get:: /api/(version)/history/events/type_mappings
+
+  Doing a GET on this endpoint will return a mapping of history events types and subtypes to the frontend's ``TransactionEventType`` representation. We also return properties such ``icon``, ``label`` and ``color`` for event types.
+
+  **Example Request**
+
+  .. http:example:: curl wget httpie python-requests
+
+    GET /history/events/type_mappings HTTP/1.1
+    Host: localhost:5042
+
+
+  **Example Response**
+
+  .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+        "result":{
+            "mappings":{
+                "spend":{
+                    "fee": "gas",
+                    "payback debt": "repay",
```

Comment:
Is there any central place where we have all "Transaction" types? The value part of this mapping? So "donate", "liquidate" etc.?